demos/theme/theme-dark.scss

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "../../packages/mdc-theme/color-palette";
+
+$mdc-theme-primary: $material-color-amber-300;
+$mdc-theme-secondary: $material-color-pink-400;
+$mdc-theme-background: $material-color-grey-900;
```

Comment:
Put a TODO here to call Sass mixins instead of using mdc-theme-primary, mdc-theme-secondary, and mdc-theme-background

Also, call the Sass mixins for the checkbox so that the demo example on theme=dark is a yellow checkbox (or pink, whatevs)

And call the Sass mixins for the radio button, the select, the tabs, and the text field (most of those mixins are written, so just use whatever is there)

Reply:
Done.

Select and Tabs don't have theme mixins yet, so I left a TODO for those.